### ISSUE TITLE
White text in matrix in dark mode?

### ISSUE THREAD

**Issue by jernst** [fediverse-devnet/feditest#206]
Was sent this screenshot. The text in the second matrix is clearly not readable.

![image](https://github.com/fediverse-devnet/feditest/assets/874785/ab3426bb-93db-4334-99d8-81607d9e6640)

**Comment by jernst** [comment]
Fixed in #221, and one more piece in #230 (not merged yet but I'll close this anyway.)
